Tools/GNUMake/packages/Make.hdf5

```diff
@@ -0,0 +1,19 @@
+CPPFLAGS += -DAMREX_USE_HDF5
+
+ifndef AMREX_HDF5_HOME
+ifdef HDF5_DIR
+  AMREX_HDF5_HOME = $(HDF5_DIR)
+endif
+ifdef HDF5_HOME
+  AMREX_HDF5_HOME = $(HDF5_HOME)
+endif
+endif
+
+LIBRARIES += -lhdf5 -lz -ldl
+
+ifdef AMREX_HDF5_HOME
+  HDF5_ABSPATH = $(abspath $(AMREX_HDF5_HOME))
+  INCLUDE_LOCATIONS += $(HDF5_ABSPATH)/include
+  LIBRARY_LOCATIONS += $(HDF5_ABSPATH)/lib
+  LIBRARIES += -L$(HDF5_ABSPATH)/lib
```

Comment:
This line is not needed because the path has already been added to `LIBRARY_LOCATIONS`.  In `Make.defs`, we have
```
LDFLAGS += -L. $(addprefix -L, $(LIBRARY_LOCATIONS))
```

Reply:
We also have:

```
ifeq ($(USE_RPATH),TRUE)
  LDFLAGS += -Xlinker -rpath -Xlinker '$(abspath .)' $(addprefix -Xlinker -rpath -Xlinker , $(abspath $(LIBRARY_LOCATIONS)))
endif
```

in Make.defs.

Reply:
Yes, but `USE_RPATH` is not defaulted to `TRUE`.